Apps/W1/ReviewGLEntries/app/src/tables/GLEntryReviewEntry.Table.al

```diff
@@ -1,7 +1,12 @@
 namespace Microsoft.Finance.GeneralLedger.Review;
 
+using Microsoft.Finance.GeneralLedger.Ledger;
+
 table 22216 "G/L Entry Review Entry"
 {
+    ObsoleteState = Pending;
+    ObsoleteReason = 'Use "G/L Entry Review Log" instead.';
+    ObsoleteTag = '27.0';
```

Comment:
Shouldn't we have an obsoleteState=removed surroundet by a IF NOT CLEAN27.
And we might need a CLEANSCHEMA around the whole table defintion.